### DISCUSSION TITLE
Google AdSense

### DISCUSSION THREAD

**Discussion by mahenrique94** [frontendbr/forum#898] [Perguntas e Respostas]
Bom dia, estou adicionando o AdSense no meu blog, porém alguns Ads ficam em vázios, apenas ocupam espaço na tela e não carregam a propaganda, o console não apresenta nenhum erro.

Alguma ajuda?

**Comment by felipefialho** [comment]
Nunca usei, mas deve ser algum tipo de autorização por parte do Google, eles já aprovaram seu cadastro?

**Comment by mahenrique94** [comment]
Sim @LFeh, alguns anúncios já funcionam, outros ficam vázios apenas ocupando o espaço do HTML, ele puxa o iframe com o body vazio.

**Comment by phenriqueleao** [comment]
Sei que pode parecer pergunta boba, mas você desabilitou o AdBlock (caso use)? Alguns meus não apareciam também e o problema era esse.

**Comment by mahenrique94** [comment]
Boa pergunta, onde eu vejo isso?

**Comment by phenriqueleao** [comment]
É uma extensão que se instala no navegador para bloquear popups, ads e propagandas do YouTube. Fica geralmente ao lado do campo de endereço do navegador.

**Comment by mahenrique94** [comment]
Entendi, nunca desabilitei não @phenriqueleao
